automation/run.ts

```diff
@@ -20,14 +20,13 @@ import * as _ from 'lodash';
 import {
 	buildOclifInstaller,
 	buildStandaloneZip,
-	fixPathForMsys,
-	ROOT,
-	runUnderMsys,
+	catchUncommitted,
 } from './build-bin';
 import {
 	release,
 	updateDescriptionOfReleasesAffectedByIssue1359,
 } from './deploy-bin';
+import { fixPathForMsys, ROOT, runUnderMsys } from './utils';
```

Comment:
These functions were refactored from `automation/build-bin.ts` to the new `automation/utils.ts`.